<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>490</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>UDP聊天室服务端</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>IP地址</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_ip">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>8</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>端口号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_port">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>8</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>8080</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>6</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开启</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>451</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>日志信息</string>
    </property>
    <widget class="QPlainTextEdit" name="te_logger">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>431</width>
       <height>161</height>
      </rect>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>451</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>在线用户</string>
    </property>
    <widget class="QTableWidget" name="tw_users">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>431</width>
       <height>121</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>用户名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>IP</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>端口</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
